.github/workflows/ci-conda-workflow.yml

```diff
@@ -4,7 +4,7 @@ on: [push, pull_request]
 
 env:
   sherpa_channel: sherpa
-  xspec_channel: "xspec/channel/test"
+  xspec_channel: "xspec/label/test"
```

Comment:
Is this an intended change or should it go back to `xspec/channel/test`?

Reply:
So, the "channel" designation is actually the old designation and "label" is the new one (for a while now) for Anaconda cloud. They are interchangeable for now, though I erred on the side of the new one in case the old one is eventually deprecated.

Reply:
Or was this a change from #1187 that is now being reverted?

Reply:
Nope. Reverting that change was removing the "_CONDA_PYTHON_SYSCONFIGDATA_NAME" variable settings.